### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by madjin**
Overall cleanup and updating of some information. Still need to add information for minting pets and mounts.

Fixes https://github.com/webaverse/docs/pull/35

### CODE REVIEW

File: docs/create/import-vroid-studio.md [hunk 1]
```diff
@@ -1,27 +1,34 @@
 ---
-id: import-vroid-studio
-title: Import From VRoid Studio
+id: import-vroid
+title: Import From VRoid
 ---
 
-This guide explains how you can export your avatar from <a href="https://vroid.com/en/studio/" target="_blank" rel="noopener noreferrer">VRoid Studio</a> to bring it into Webaverse.
+Webaverse uses [VRM](https://vrm.dev/en/), a 3D avatar file format based on the royalty free glTF2.0 standard. VRM avatars have features such as blendshapes for emotes, facial animations, physics, and are widely supported across dozens of apps and VR platforms.
```
Comment (avaer) [inline docs/create/import-vroid-studio.md:6]: I wouldn't say royalty-free; should emphasize the features of the format like in the second sentence first.

Comment (avaer) [reply docs/create/import-vroid-studio.md:6]: But it's fine either way, just style.

File: docs/create/xrpackage-overview.md [hunk 2]
```diff
@@ -7,7 +7,9 @@ title: What is XRPackage?
 
 XRPackage turns 3D apps into a file you can load anywhere.
 
-It uses standards like WebGL, WebXR, GLTF, and WebBundle to package an app into a `.wbn` file.
+![Moving an XRPK around in a scene](/img/xrpks.gif)
+
+It uses standards like WebGL, WebXR, GLTF, and WebBundle to package an app into a `.wbn` file. The Github repo for XRPackages is here: https://github.com/webaverse/xrpackage
```
Comment (avaer) [inline docs/create/xrpackage-overview.md:12]: Nah I wouldn't link out to https://github.com/webaverse/xrpackage

File: docs/webaverse/nfts.md [hunk 3]
```diff
@@ -4,31 +4,44 @@ title: NFTs
...
-You can access all of the digital assets you own in Webaverse in your inventory.
+
+You can access all of the digital assets you own in Webaverse in your inventory a couple different ways. From Discord you can type `.inventory` in a server the [Discord bot](https://webaverse.com/discordbot) is installed in. You can also go to the [main website](https://webaverse.com/) and click the icon on the top right to see your profile page that has your inventory there.
   
 ![profile inventory](/img/inventory.png)
 
 ## SILK vs NFTs
-SILK is fungible token, which means any one SILK is equivalent to another. (think of dollars)
+
+SILK is fungible token, which means any one SILK is equivalent to another. Can think of currency.
```
Comment (avaer) [inline docs/webaverse/nfts.md:32]: Not a full sentence.

File: docs/webaverse/nfts.md [hunk 4]
```diff
@@ -4,31 +4,44 @@ title: NFTs
...
   
 ![fungible token example](/img/equalft.png)
   
-On the other hand, NFTs are non-fungible which means, no two NFTs are the same. (think different kinds of pokemon cards)
+On the other hand, NFTs are non-fungible which means, no two NFTs are the same. Unique assets such as the the Mona Lisa or land is considered non-fungible.
```
Comment (avaer) [inline docs/webaverse/nfts.md:36]: - the the
- are considered

File: docs/webaverse/parcels.md [hunk 5]
```diff
@@ -3,6 +3,8 @@ id: parcels
 title: Parcels 
 ---
 
-Parcels represent an area of the Webaverse world that you can own and build whatever you wish on.
+Parcels represent an area of the Webaverse world that you can own and build whatever you wish on. Size is not a limitation, you can build full virtual experiences that can be hyperlinked to The Street.
```
Comment (avaer) [inline docs/webaverse/parcels.md:6]: from The Street

File: docs/create/mounts.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
```
Comment (avaer) [inline docs/create/mounts.md:1]: These need to be filled in. There are new NFT types coming too, such as shaders.

**Review by avaer — COMMENTED** [review]
Some small things, but overall a good improvement.

File: docs/webaverse/nfts.md (hunk 3 above)
Comment (madjin) [reply docs/webaverse/nfts.md:32]: Changed to:
> SILK is a fungible token which means that any one SILK is equivalent to another. You can think of currency or grains of rice as an example of something that's fungible.

File: docs/webaverse/parcels.md (hunk 5 above)
Comment (madjin) [reply docs/webaverse/parcels.md:6]: fixed

File: docs/create/import-vroid-studio.md (hunk 1 above)
Comment (avaer) [inline docs/create/import-vroid-studio.md:6]: Maybe worth mentioning them?

File: docs/create/mint.md [hunk 7]
```diff
@@ -3,8 +3,26 @@ id: mint
...
+
+Drag and drop a file into a server that contains the [Discord bot](https://webaverse.com/discordbot) and type `.mint` as the message. If you want to mint more than 1 edition, type `.mint 5` or however many in the message field.
+
+![Minting an image NFT in Discord](/img/mint.jpg)
+
+More: [How To Mint NFTs in Discord](/docs/create/mint-discord)
+
+### Twitter
+
+Coming soon
```
Comment (avaer) [inline docs/create/mint.md:28]: Should be documented.

File: docs/webaverse/.vs/VSWorkspaceState.json [hunk 8]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (avaer) [inline docs/webaverse/.vs/VSWorkspaceState.json:1]: Do we need visual studio stuff here?

File: docs/webaverse/discord-bot.md [hunk 9]
```diff
@@ -3,23 +3,22 @@ id: discord-bot
...
 ### `.help`
 show help
 
-## Account 
-
 ### `.status`
 show your account details
 
-### `.balance` 
+### `.balance [@user|0xaddr]`
```
Comment (avaer) [inline docs/webaverse/discord-bot.md:34]: Should add a question mark here to signify that the argument is optional.

File: docs/create/xrpackage-overview.md [hunk 10]
```diff
@@ -5,7 +5,9 @@ title: What is XRPackage?
 
 <img src="/img/xrpk-logo.png" width="250px" height="250px" />
 
-XRPackage turns 3D apps into a file you can load anywhere.
+[XRPackage](https://github.com/webaverse/xrpackage) turns 3D apps into a file you can load anywhere.
```
Comment (avaer) [inline docs/create/xrpackage-overview.md:8]: Don't link it, it will be confusing.

File: docs/webaverse/discord-bot.md [hunk 11]
```diff
@@ -80,9 +80,36 @@ get metadata key [key] for NFT [id]
...
 
+### `.uncollab [@user|0xaddr] [tokenId]`
+remove collaborator from [tokenId]
+
+---
+
+## Account 
+
+### `.name [newname]`
+set your name to [name]
```
Comment (avaer) [inline docs/webaverse/discord-bot.md:94]: `newname`?

File: docs/webaverse/nfts.md [hunk 12]
```diff
@@ -38,4 +52,12 @@ Non-fungible means that it is not interchangable, so while you can trade NFTs, y
...
-If you want to transfer your SILK or NFTs to the main Ethereum change, you can do so. The ability to move to the Ethereum blockchain also allows you to sell your NFTs on a marketplace like OpenSea.
+If you want to transfer your SILK or NFTs to the main Ethereum chain, you can do so from the ASSET item page on the website. Note: the user has to pay a gas transaction fee to do the transfer.
+
+![Transfer to mainnet](/img/xfermainnet.jpg)
+
+The ability to move to the Ethereum blockchain also allows you to sell your NFTs on a marketplace like OpenSea.
+
+### What happens when I mint a NFT?
+
+When you mint a file from the web interface or Discord bot it gets automatically uploaded to [IPFS](https://ipfs.io/) with data about the token uploaded to the Webaverse sidechain. This process takes 0 gas because its all on the [sidechain](https://ethereum.org/en/developers/docs/layer-2-scaling/#sidechains), a separate blockchain which runs in parallel to mainnet and operates independently. You can choose to then move it to mainnet afterwords, just pay the transaction fees.
```
Comment (avaer) [inline docs/webaverse/nfts.md:63]: > afterwords
afterwards

Also it doesn't take "0 gas", it's more like gas is free.

File: docs/webaverse/parcels.md [hunk 13]
```diff
@@ -3,6 +3,8 @@ id: parcels
 title: Parcels 
 ---
 
-Parcels represent an area of the Webaverse world that you can own and build whatever you wish on.
+Parcels represent an area of the Webaverse world that you can own and build whatever you wish on. Size is not a limitation, you can build full virtual experiences that can be hyperlinked from The Street.
```
Comment (avaer) [inline docs/webaverse/parcels.md:6]: The Street needs to be a `.md`. Some details are in `seasons.md` but we need a much higher level and image-filled explainer of the street concept.

File: docs/webaverse/silk.md [hunk 14]
```diff
@@ -0,0 +1,24 @@
...
+
+SILK is a token that is used in Webaverse primarily to mint [NFTs](./nfts). It's main purpose is to invite others into the network and prevent spam so together we can build a web of trust as the Webaverse scales.
+
+If you're a Discord server admin that wants to distribute SILK amongst your community to use the [NFT Discord Bot](https://webaverse.com/discordbot) then ping us on [Discord](https://discord.gg/3byWubumSa) or [email us](hello@webaverse.com).
+
+## FAQ
+
+### Why the name SILK?
+
+Backstory: For years we've been thinking about and working on interoperability problems in the space. We want projects to be more compatible with one another so we can have a Metaverse or 3D internet. Right now virtual worlds act as separate colonies or countries, each with their own currencies and moderation practices.
```
Comment (avaer) [inline docs/webaverse/silk.md:14]: I'd skip this and just use the second paragraph.

Comment (avaer) [reply docs/webaverse/silk.md:14]: If we want to talk about ourselves, we should do it as a footnote to the point that we are really trying to make.

File: docs/create/import-vroid-studio.md (hunk 1 above)
Comment (madjin) [reply docs/create/import-vroid-studio.md]: added some:

- [Cluster](https://cluster.mu/) - PCVR, Windows, macOS, iOS, Android
- [Craftopia](https://store.steampowered.com/app/1307550/Craftopia/) - Steam
- [VRAST!](https://rc-cobalt.booth.pm/items/1756359) - Windows
- [VseeFace](https://www.vseeface.icu/) - Windows
- and more: https://vrm.dev/en/vrm_applications/

File: docs/webaverse/discord-bot.md (hunk 9 above)
Comment (madjin) [reply docs/webaverse/discord-bot.md:34]: added missing question marks

File: docs/create/xrpackage-overview.md (hunk 10 above)
Comment (madjin) [reply docs/create/xrpackage-overview.md:8]: fixed

File: docs/webaverse/nfts.md (hunk 12 above)
Comment (madjin) [reply docs/webaverse/nfts.md:63]: fixed

File: docs/webaverse/silk.md (hunk 14 above)
Comment (madjin) [reply docs/webaverse/silk.md:14]: gunna leave it out for now

File: docs/webaverse/.vs/VSWorkspaceState.json (hunk 8 above)
Comment (madjin) [reply docs/webaverse/.vs/VSWorkspaceState.json:1]: don't think so, wonder if I could just delete the .vs folder

**Pull request merged**